pkg/monitor/monitor/util.go

```diff
@@ -687,7 +697,6 @@ func getMonitorPrometheusReloaderContainer(monitor *v1alpha1.TidbMonitor) core.C
 			"--reload-url=http://localhost:9090/-/reload",
 			"--config-file=/etc/prometheus/config/prometheus.yml",
 			"--config-envsubst-file=/etc/prometheus/config_out/prometheus.yml",
-			"--watched-dir=/etc/prometheus/config/prometheus.yml",
```

Comment:
No need to watch the dir of prom config?

Reply:
We use prom reloader,it will add prom config automatically.
![image](https://user-images.githubusercontent.com/13257127/132460009-20807915-d506-4097-a8ae-68e7b27fa753.png)